specification/redisenterprise/resource-manager/Microsoft.Cache/stable/2022-01-01/redisenterprise.json

```diff
@@ -1763,6 +1805,133 @@
         "ids"
       ],
       "description": "Parameters for a Redis Enterprise Active Geo Replication Force Unlink operation."
+    },
+    "RegionSkuDetail": {
+      "title": "Region Sku Detail",
+      "description": "Details about the location requested and the available skus in the location",
+      "type": "object",
+      "properties": {
+        "resourceType": {
+          "type": "string"
```

Comment:
Add description please, like 'Resource type which has the SKU, such as Microsoft.Cache/redisEnterprise'